hydra-node/src/Hydra/Options.hs

```diff
@@ -106,9 +110,25 @@ networkMagicParser =
       ( long "network-magic"
           <> metavar "MAGIC"
           <> value 42
-          <> help "Network magic for the target network."
+          <> help "Network magic for the target network (default: 42)."
       )
 
+nodeSocketParser :: Parser FilePath
+nodeSocketParser =
+  strOption
+    ( long "node-socket"
+        <> metavar "FILE"
+        <> help "Local (Unix) socket path to connect to cardano node."
+    )
+
+pathToWalletKeyParser :: Parser FilePath
+pathToWalletKeyParser =
+  strOption
+    ( long "wallet-key-file"
```

Comment:
May I suggest: `wallet-signing-key`

- "file" is already hinted by the metavar
- there are, at least in the context of the wallet, two keys, so it's good to say which one.